enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
+    - groupName: uat:
+      rolloutApproved: false
+    - groupName: prod
+      rolloutApproved: false
+  ungrouped:
+    rolloutApproved: false
+```
 
-  (For technical notes see [Development Phase 1](#development-phase-1))
+This control is separate from the policy's `spec.rolloutStrategy` because the `status` field is required for the
+technical implementation as described in another section. Having it separate also allows control of the rollout in a
+console or CLI even if the policy is managed by GitOps. Additionally, if the policy applies to multiple hubs, the groups
+may be different on those hubs.
+
+The name, with a `policy-` prefix, and the namespace of the `Rollout` needs to be the same as the root policy it refers
+to. This object will be automatically created by the Governance Policy Propagator with only the `status` and
+`metadata.ownerReferences` fields set for storing state.
+
+The `spec.decisionGroups` field is set by the user to indicate which groups are approved. These groups directly match
+those in the [decisionStrategy](https://open-cluster-management.io/concepts/placement/#decision-strategy) field in the
+`Placement` API. If the user wants a subset of the groups to be automatically approved, they would add the groups to the
+policy's `spec.rolloutStrategy.manualPerGroup.mandatoryDecisionGroups` array.
+
+If the user wants the approval to be conditional on a policy version, the user must set `spec.approvalsForVersion` to a
+value that matches the `policy.open-cluster-management.io/version` annotation on the root policy. By default, the
+approvals apply to all policy versions.
+
+If the decision groups do not capture all clusters in the placement predicate, the user can set
+`spec.ungrouped.rolloutApproved` to `true` to approve those clusters.
+
+### Status
+
+#### Replicated Policy Status
+
+A replicated policy is the policy copy in the managed cluster namespace. Each replicated policy will have a rollout
+status defined as follows:
+
+| Status          | Description                                                                                           |
+| --------------- | ----------------------------------------------------------------------------------------------------- |
+| `ToApply`       | The cluster is waiting for the new version of the policy but the old version is still applied.        |
+| `Progressing`   | The new version of the policy has been deployed but the rollout status is not yet known.              |
+| `Succeeded`     | The policy has confirmed it is compliant.                                                             |
+| `Failed`        | The policy is still noncompliant after `progressDeadline` has elapsed.                                |
+| `TimeOut`       | Time has passed beyond the timeout specified in `spec.rolloutStrategy` without a returned compliance. |
+| `NotApplicable` | The policy was applied to the cluster after the rollout completed.                                    |
```

Comment:
I get that this new status is trying to say it's N/A for the rollout, but that might be confusing to me as a user to see an N/A status and understanding what that means. It seems to me adding a new cluster and then having it fail would also use these other statuses because the rollout (even if the original rollout had ended long ago) has now failed.

Like if I added an entire batch of clusters that were spread across groups, I think I'd want to see whether that "mini" rollout had succeeded.

Reply:
I'm trying to limit the rollout to new policy versions rather than new clusters to make it conceptually simpler for the user and for implementation.

For your scenario, I think the workaround would be to set the policy as inform and then flip it to enforce after the clusters have been added. Then a new rollout would trigger since the generation of the policy changed.

Reply:
In that case, maybe the status should be "NewCluster". I think that'd make more sense to me.